rootfs/scheduler/__init__.py

```diff
@@ -936,62 +936,6 @@ def delete_rc(self, namespace, name):
 
         return response
 
-    def _healthcheck(self, namespace, container, routable=False, path='/', port=5000,
```

Comment:
Yeah, that was **one** of the things rattling around. Was also looking for other ideas if you see a better way :) A bit of pre-caffeine rubber ducking.

If we moved it over then I wouldn't want to lose the error message handling in the Serializer. It's a little funky when we have such big scope shared in two places but the nature of the beast I guess :)